skalvi/views.py

```diff
@@ -100,6 +100,18 @@ def signOfEvent(request):
         # Will never happen
         return HttpResponse(status=206)  # not logged in
 
+def getAttendingActivities(request):
```

Comment:
extract to skalvi/ApiFunctions.py and then in urls.py where u use it, change to ApiFunctions.getAttendingActivities ?